source/includes/_queries.md

```diff
@@ -287,6 +288,58 @@ Field | Description | Constraint
 -------------- | -------------- | --------------
 Transactions | an array of transactions for given account and asset | Committed transactions
 
+## Get transactions
+
+### Purpose
+
+When an information about transaction per its transaction's hash is needed, `GetTransactions` query is sent.
+
+Given this Query, in successful case Iroha returns `TransactionsResponse`, which is a collection of transactions.
+
+### Structure
+
+#### Query
+
+> Query
+
+```protobuf
+message GetTransactions {
+    repeated bytes tx_hashes = 1;
+}
+```
+```json
+{
+    "signature":
+        {
+            "pubkey": "…",
+            "signature": "…"
+        },
+    "created_ts": …,
+    "creator_account_id": "admin@test",
+    "query_counter": 1,
+    "query_type" : "GetTransactions",
+    "tx_hashes": [string(64),…]
+}
+```
+
+Field | Description | Constraint
+-------------- | -------------- | --------------
+Transactions hashes | transactions' hashes to request transactions from  | (proto) array of 32 bytes, (json) 64 size of hex strings
```

Comment:
Imo we should follow in the field the exact name of the proto's message, ping @neewy 
Though out of this pr scope